libs/core/algorithms/tests/regressions/for_loop_test.cpp

```diff
@@ -0,0 +1,26 @@
+#include <hpx/local/algorithm.hpp>
```

Comment:
Please add the Boost license and you copyright to this file (as per https://github.com/STEllAR-GROUP/hpx/blob/master/.github/CONTRIBUTING.md)

Reply:
Sure

Reply:
@hkaiser should I have to write my copyright in for_loop.hpp file also

Reply:
> @hkaiser should I have to write my copyright in for_loop.hpp file also

The rule of thumb we apply is to add your copyright to a file if you have contributed/touched on ~10-15% of the code in that file.

Reply:
Got it , thanks for explaining.